docs/Home.md

```diff
@@ -1,21 +1,23 @@
 ### What is splat?
 
-**splat** is a binary splitting tool, written in Python.
+**splat** is a binary splitting tool, written in Python. It's goal is to support the successful disassembly and then reassembly of binary data.
```

Comment:
To me, "reassembly" implies it only supports compiling assembly files only, not covering compiling C files too

Reply:
yeah, perhaps "rebuilding"

Reply:
also "Its"